setup.py

```diff
@@ -43,13 +43,13 @@
 
     with open(fname, 'w') as f:
         nchars = len(gitVersion) - 1
-        f.write("__gitPath__='{0}'\n".format(gitPath))
-        f.write("__gitVersion__='{0}'\n".format(gitVersion[:nchars]))
+        f.write("__gitPath__ = '{0}'\n".format(gitPath))
+        f.write("__gitVersion__ = '{0}'\n".format(gitVersion[:nchars]))
```

Comment:
Future improvement to use the `pygit2` lib and move this to the common package as well. Seen this throughout the NWRC setup.py

Reply:
We use this in all the major packages. It would be nice to have this abstracted to limit code repeats. We have done this with `spatialnc` for some of the topo functions but not general functions.